src/compiler/compile.js

```diff
@@ -9,7 +9,8 @@ import {getBuilders} from './builders';
  * @return {import('./types').CompilerResponse}
  */
 export function compile(request) {
-  const {document, nodes, versions} = request ?? {};
+  // eslint-disable-next-line local/camelcase
+  const {component_versions: versions, document, nodes} = request ?? {};
```

Comment:
```suggestion
  const {'component_versions': versions, document, nodes} = request ?? {};
```

Reply:
I think I slightly prefer the comment... 🤔. Feels weirder to counter-intuitively change the source to conform to a lint rule than to explicitly violate the rule but mark it as OK

Reply:
Interesting, my take is the complete opposite; the point of eslint being that it shifts us to write code in ways that don't require exception comments but instead conform to its style preferences (as defined by us through config)

Reply:
imo almost any eslint-disable is a code smell

Reply:
what about this specific case? where we have no choice. the offending code is outside of our codebase...and we need to handle it somehow.

Reply:
Maybe the eslint rule should only apply to object creation?

Reply:
I'll deploy this fix now, lets bikeshed over a call 🥇

Reply:
I think it's really only relevant specifically in this kind of destructuring case. `const versions = request.component_versions` wouldn't complain. Not sure how to fix

Reply:
Also is a problem with constructing objects meant for 3p libraries, like `terser`

https://github.com/ampproject/amphtml/blob/abfe09e34b17a53fc676f9f2ae98ad8065fd5d77/build-system/tasks/helpers.js#L535-L542

Reply:
Quoting the property works here as well

Reply:
I see it as comparable to how we sometimes do `[SomethingEnum.Value]: "blah"`

Reply:
The original eslint rule hasn't been updated in years. We can add support for destructuring, since the rule is specifically not supposed to complain on access, only at definition.